apps/src/p5lab/spritelab/libraries/libraryFactory.js

```diff
@@ -0,0 +1,10 @@
+import CoreLibrary from './CoreLibrary';
+import PoemBotLibrary from './PoemBotLibrary';
+
+export default function getLibrary(levelConfig, p5) {
```

Comment:
a few of nits:

this function might be more appropriate as `createLibrary` since it's creating the library you need. (i think my thoughts on this are also biased because the factory pattern is a type of "creation" pattern.)

this factory will be more flexible if it accepts an object as a single argument -- as we add new library types, they will likely require new params, but other existing callers may not care about those new params.